spec/acceptance/suites/default/10_one_node_spec.rb

```diff
@@ -25,12 +25,12 @@
 
         on(host, 'yum install -y epel-release', run_in_parallel: true)
 
-        set_hieradata_on(host, { 'simp_options::firewall' => true })
```

Comment:
why did you remove this?

Reply:
It used to be needed for the parameter I just removed